### PR TITLE
Add keyboard Ergo68

### PR DESCRIPTION
**Pull request by MakotoKurauchi**
## Description

Add keyboard Ergo68

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/yushakobo/ergo68/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Bootloader
+
+Enter the bootloader in 3 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
+* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (zvecr) [inline keyboards/yushakobo/ergo68/readme.md:23]:
```suggestion
Flashing example for this keyboard:

    make yushakobo/ergo68:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

File: keyboards/yushakobo/ergo68/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    ),
+        [3] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS
+    )
```
Comment (zvecr) [inline keyboards/yushakobo/ergo68/keymaps/via/keymap.c:47]:
```suggestion
    )
```

File: keyboards/yushakobo/ergo68/config.h
Comment (zvecr) [inline keyboards/yushakobo/ergo68/config.h:1]: This file should be removed, as all the contents can be migrated to `info.json`.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/yushakobo/ergo68/info.json [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        "mousekey": true,
+        "rgb_matrix": true,
+        "nkro": false
+    },
+    "matrix_pins": {
+        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2"],
+        "rows": ["D4", "C6", "D7", "E6", "B4"]
+    },
+    "debounce": 5,
+    "processor": "atmega32u4",
```
Comment (dunk2k) [inline keyboards/yushakobo/ergo68/info.json:21]: 1. Pro micro mentioned in readme.md and confirmed in build guide
2. `5` is default value for debounce
3. removed settings in `features` that are disabled/false by default
```suggestion
    "features": {
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true,
        "rgb_matrix": true,
    },
    "development_board": "promicro",
    "diode_direction": "COL2ROW",
    "matrix_pins": {
        "cols": ["F4", "F5", "F6", "F7", "B1", "B3", "B2"],
        "rows": ["D4", "C6", "D7", "E6", "B4"]
    },
```

File: keyboards/yushakobo/ergo68/keymaps/via/rules.mk [hunk 4]
```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```
Comment (dunk2k) [inline keyboards/yushakobo/ergo68/keymaps/via/rules.mk:1]:
```suggestion
VIA_ENABLE = yes

```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/yushakobo/ergo68/config.h
Comment (MakotoKurauchi) [reply keyboards/yushakobo/ergo68/config.h:1]: ok

**Comment by MakotoKurauchi** [comment]
Thanks for the suggestion. Please check.

File: keyboards/yushakobo/ergo68/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_PSCR,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T, KC_LBRC, KC_RBRC,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G, KC_MINS,  KC_EQL,    KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN,  KC_ENT,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,  KC_DEL, KC_BSPC,    KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_RSFT,
+        KC_LCTL,KC_LGUI,KC_LALT,  MO(1),  KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC, KC_RALT,  KC_APP, KC_LCTL
+    ),
+        [1] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
```
Comment (dunk2k) [inline keyboards/yushakobo/ergo68/keymaps/default/keymap.c:28]: makes layer 1 functional
```suggestion
        QK_BOOT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
```

File: keyboards/yushakobo/ergo68/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0, KC_PSCR,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T, KC_LBRC, KC_RBRC,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G, KC_MINS,  KC_EQL,    KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN,  KC_ENT,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,  KC_DEL, KC_BSPC,    KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_RSFT,
+        KC_LCTL,KC_LGUI,KC_LALT,  MO(1),  KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC,  KC_SPC, KC_RALT,  KC_APP, KC_LCTL
+    ),
+        [1] = LAYOUT(
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
```
Comment (dunk2k) [inline keyboards/yushakobo/ergo68/keymaps/via/keymap.c:28]: makes layer 1 functional
```suggestion
        QK_BOOT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/yushakobo/ergo68/info.json [hunk 7]
```diff
@@ -0,0 +1,227 @@
...
+            {"matrix": [8, 0], "x": 224, "y": 45, "flags": 1},
+            {"matrix": [9, 0], "x": 224, "y": 57, "flags": 1},
+            {"matrix": [9, 1], "x": 210, "y": 57, "flags": 1},
+            {"matrix": [9, 2], "x": 196, "y": 54, "flags": 1},
+            {"matrix": [9, 3], "x": 175, "y": 54, "flags": 1},
+            {"matrix": [9, 4], "x": 161, "y": 55, "flags": 1},
+            {"matrix": [9, 5], "x": 144, "y": 58, "flags": 1},
+            {"matrix": [9, 6], "x": 130, "y": 64, "flags": 1}
+        ],
+        "led_flush_limit": 16,
```
Comment (waffle87) [inline keyboards/yushakobo/ergo68/info.json:104]:
```suggestion
```

File: keyboards/yushakobo/ergo68/info.json [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+        },
+        "led_process_limit":16
+    },
+    "split": {
+        "enabled": true,
+        "handedness": {
+            "pin": "B6"
+        },
+        "transport": {
+            "protocol": "serial",
```
Comment (waffle87) [inline keyboards/yushakobo/ergo68/info.json:135]:
```suggestion
```

**Review by waffle87 — APPROVED** [review]

File: keyboards/yushakobo/ergo68/info.json [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+            "raindrops": true,
+            "solid_reactive": true,
+            "splash": true,
+            "solid_splash": true,
+        },
+        "default":{
+          "hue": 0,
+          "sat": 255,
+          "val": 50,
+          "speed": 127
```
Comment (zvecr) [inline keyboards/yushakobo/ergo68/info.json:124]: Should avoid duplication of default values
```suggestion
          "val": 50
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**